modules/ichorcna/1.0/ichorcna.smk

```diff
@@ -70,7 +70,7 @@ rule _install_ichorcna:
         CFG["conda_envs"]["ichorcna"]
     shell:
         op.as_one_line("""
-        git clone git@github.com:broadinstitute/ichorCNA.git {params.outdir} &&            
+        git clone git://github.com/broadinstitute/ichorCNA.git {params.outdir} &&            
```

Comment:
Shouldn't this be:
git clone https://github.com/broadinstitute/ichorCNA.git 
?